packages/common/docs/pipes.md

```diff
@@ -749,3 +749,25 @@ export class AppComponent {
     public someArray = ['first', 'second', 'third', 'last'];
 }
 ```
+
+-   `ToBooleanPipe, ToBooleanPipeModule`
+
+```ts
+import { ToBooleanPipeModule } from '@angular-ru/common/pipes';
+
+@NgModule({
+    // ..
+    imports: [ToBooleanPipeModule]
+})
+export class AppModule {}
+
+@Component({
+    //...
+    template: `
+        <mat-select [multiple]="multiple | toBoolean">first</p>
```

Comment:
maybe call it `coerceBoolean`, as internal function?

Reply:
![image](https://user-images.githubusercontent.com/12021443/121010749-782e6a00-c79e-11eb-9fc5-99a33c468fc1.png)

Maybe, but I think we should align naming with another converters

Reply:
i mean, coerceBoolean and toBoolean is different things. 
`toBoolean = val => Boolean(val)`
`coerceBoolean = val => val !== '' ...`

but if you want to forgive the dumb logic of falsy values, i will support it.

Reply:
renamed